src/routes/reference/pca.svx

```diff
@@ -12,4 +12,45 @@ flair: reference
 category: Analyse Data
 ---
 
+<script>
+    import YouTube from '$lib/components/YouTube.svelte';
+    import ResourceLink from '$lib/components/ResourceLink.svelte';
+</script>
 
+Principal Component Analysis (PCA) fits to a [DataSet](/reference/dataset) to determine its _principal components_, each of which is a new axis through the data that maximises the variance, or "differences", within the Data. PCA can then transform the original DataSet or individual points to position them in relation to the principal components (i.e., "new axes") for better comparing how they differ from other points in the DataSet. PCA is often used for dimensionality reduction (see below) and is also useful for removing redundancy (i.e., correlation) and/or noise (i.e., dimensions that are [uniformly distributed](/learn/distribution)) from a DataSet.
+
+For a great visual explanation of PCA, watch this Computerphile video:
+
+<YouTube 
+url={'TJdH6rPA-TI'}
+/>
+
+## Calculating the Principal Components
+
+In order to calculate the principal components, PCA analyses the original DataSet and determines how to "rotate" the data so that an axis can be drawn through the data along which the most variance in the data can be observed (for a excellent visual explanation, see the YouTube video above). The new axis, or attribute, is a principal component of the DataSet. PCA continues the process of "drawing axes", through the data (always maximising variance) until all the principal components have been determined (PCA computes as many principal components as there are dimensions in the original DataSet). This process orders the principal components (starting at 1) according to how much of the variance in the DataSet they are able to demonstrate (from demonstrating the most variance to the least). The lower ordered principal components are therefore _more useful_ in understanding the differences between the points in the DataSet.
+
+## PCA for Dimensionality Reduction
+
+Because PCA orders the principal components from those that demonstrate the most variance in the data to the least, one can effectively perform dimensionality reduction by removing some of the higher ordered principal components (in the transformed data set) because they are less useful at representing the variance in the data. This allows the data set to have fewer dimensions, while retaining the aspects of the data that are more useful in understanding the differences between the data points (the lower order principal components). 
```

Comment:
Reading so far there is a lot of caveated bracketed writing which I find bumpy. I think it doesn't need to be expressed in this way. For example:

> In order to calculate the principal components, PCA analyses the original DataSet and determines how to "rotate" the data so that an axis can be drawn through the data along which the most variance in the data can be observed (for a excellent visual explanation, see the YouTube video above)

This is just two sentences. Furthermore we should be linking to a timestamp.

> PCA continues the process of "drawing axes", through the data (always maximising variance) until all the principal components have been determined (PCA computes as many principal components as there are dimensions in the original DataSet)

PCA continues the process of "drawing axes" through the data until all the principal components have been determined. It consistently aims to maximise the variance of these axes and will compute as many principal components as there are dimensions in the original DataSet.

> This process orders the principal components (starting at 1) according to how much of the variance in the DataSet they are able to demonstrate (from demonstrating the most variance to the least)

This process orders the principal components according to how much of the variance in the DataSet they are able to demonstrate. In other words, the first principal component demonstrates the most amount compared to the last.